src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+const BigNum = require('big.js');
```

Comment:
You are using the incorrect npm module, the one that is required in the [Milestone Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896094/) is [this one](https://www.npmjs.com/package/big-js), big-js not big.js. Change that and let's get back on using import.